### DISCUSSION TITLE
Duplicate windows and tabs

### DISCUSSION THREAD

**Discussion by raptorjr** [sienori/Tab-Session-Manager#1143] [General]
Hello

I have two Chrome windows. Every time I start the computer one of the windows is always duplicated and in the other window all tabs are always duplicated. Is there some setting I can use to avoid this or is it a bug?

**Comment by petro604** [comment]
I encountered something similar with Firefox after setting Startup Behaviour to "open previous session".  Firefox already has this feature built-in and was enabled.  So they both tried to restore all the windows and tabs causing duplicates of everything.
